app/Console/Kernel.php

```diff
@@ -15,7 +15,7 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        // $schedule->command('inspire')->hourly();
+        $schedule->command('nbn:dispatch-orders')->everyFiveMinutes()->withoutOverlapping();
```

Comment:
Scheduler uses `withoutOverlapping()` to avoid duplicate dispatch if runs overlap.